kubernetes-model/pkg/schemagen/generate.go

```diff
@@ -541,3 +553,22 @@ func (g *schemaGenerator) addConstraints(objectName string, propName string, pro
 func pkgPath(t reflect.Type) string {
 	return strings.TrimPrefix(t.PkgPath(), "github.com/fabric8io/kubernetes-client/kubernetes-model/vendor/")
 }
+
+func (g *schemaGenerator) crdScope(t reflect.Type) CrdScope {
+	switch t.Name() {
+	case "MutatingWebhookConfiguration",
+		"ValidatingWebhookConfiguration",
+		"CustomResourceDefinition",
+		"ComponentStatus",
+		"Namespace",
+		"Node",
+		"PersistentVolume",
+		"PodSecurityPolicy",
+		"ClusterRoleBinding",
+		"ClusterRole",
+		"PriorityClass",
+		"StorageClass":
+		return Cluster
+	}
+	return Namespaced
```

Comment:
```suggestion
	switch t.Name() {
	case "MutatingWebhookConfiguration",
		"ValidatingWebhookConfiguration",
		"CustomResourceDefinition",
		"ComponentStatus",
		"Namespace",
		"Node",
		"PersistentVolume",
		"PodSecurityPolicy",
		"ClusterRoleBinding",
		"ClusterRole",
		"PriorityClass",
		"StorageClass":
            return Cluster
	default:
	    return Namespaced    
	}
```

Reply:
What's the benefit of this change?

Reply:
I think you're right. Switch case without `default` is generally considered code smells. let me modify this too in the next update.